docker-compose.yml

```
@@ -0,0 +1,10 @@
+services:
+  trash-searcher:
+    build:
+      context: .
+      dockerfile: distribution/Dockerfile
+    container_name: trash-searcher
+    working_dir: /app
+    volumes:
+      - ./src:/app
+    restart: on-failure
```

можно сразу добавить переменные среды
tty: true - позволяет не выключать контейнер после завершения скрипта
```suggestion
    restart: on-failure
    env_file: .env
    tty: true
```